### DISCUSSION TITLE
I cant use app commands in discord

### DISCUSSION THREAD

**Discussion by rakaxlfux** [Rapptz/discord.py#9741] [Q&A]
![image](https://github.com/Rapptz/discord.py/assets/159986888/04ab0167-f9d2-4371-ada3-3916217c7dfb)
its still not showing anything. please the helps!
it started this when i installed discord.js

**Comment by itswilliboy** [comment]
Did you try restarting Discord with `Ctrl` + `R`?

**Reply by rakaxlfux** [reply]
still doesnt work, sorry

**Comment by No767** [comment]
@rakaxlfux This is normal behavior if you are referring to having no app commands for your bot. You haven't synced these commands to Discord yet
